fmpz.h

```diff
@@ -606,6 +608,9 @@ FLINT_DLL void fmpz_mod(fmpz_t f, const fmpz_t g, const fmpz_t h);
 
 FLINT_DLL void fmpz_smod(fmpz_t f, const fmpz_t g, const fmpz_t h);
 
+FLINT_DLL void _fmpz_mods(fmpz_t r, const fmpz_t a, const fmpz_t m,
```

Comment:
Docs missing. Seem awfully easy to get mixed up with fmpz_smod. Is there another name we could use. Is it even worth exporting to the user?

Reply:
How do I use it internally without exporting it to the user?

Reply:
If it is used in multiple files, then it's fine. If it is used in a single file that's a different story.

Reply:
Feel free to change the name to what ever you think is best. I need such a function as this. Maybe we could call it _fmpz_smod, and put it as a helper function with fmpz_smod.

Reply:
If it is symmetric mod, then yes, please, do that.